portfolio/src/main/webapp/quiz.js

```diff
@@ -0,0 +1,58 @@
+
+
+function makeQuiz(){
+    questions.forEach((currentQuestion,questionNumber)=>{
+        output.push(
+            `<input type="checkbox" id="question${questionNumber}" name="question${questionNumber}" 
+            value="yes">
+            <label for="question${questionNumber}">${currentQuestion}</label></br>`);
+    });
+    quizContainer.innerHTML=output.join('');
+}
+function showResults(){
+    let count = 0;
+    questions.forEach( (currentQuestion, questionNumber) => {
+        let currentValue = document.getElementById(`question${questionNumber}`);
```

Comment:
[Prefer `const` for values that don't change.](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)